### ISSUE TITLE
Linux and OS X support

### ISSUE THREAD

**Issue by gsedej** [luciusDXL/XL-Engine#16]
I would just like to add issue to support linux (also to show interest in project).
Know you mentioned it will take some time.

**Comment by psi29a** [comment]
It was mentioned that it _would_ happen after Beta was released. :)

**Comment by luciusDXL** [comment]
Makes sense. I added OS X support to the ticket since adding it (after having Linux support) is pretty straight forward, at least for an application like this one that makes very little use of the system GUI.

**Comment by gsedej** [comment]
I can do testing after initial linux support. Also OpenMW community would probably be very interested once it linux is an option, since most of development is done on linux (IIRC).

**Comment by psi29a** [comment]
@gsedej that would be I (in reference to OpenMW)... ;)

**Comment by 7oxicshadow** [comment]
+1 for Linux support. Looking forward to playing Dark Forces again. :)

**Comment by Kasijjuf** [comment]
+1 for GNU/Linux

**Comment by JeremyHargis** [comment]
Yes, so exciting! Great to see this under development again.
